connector_test.go

```diff
@@ -586,20 +555,13 @@ func testNetworking(t *testing.T, podmanNetworking string, containerTest string,
 		readBuffer = readOutputUntil(t, plugin, *mac)
 	}
 	logger.Infof(string(readBuffer))
-	if len(readBuffer) == 0 {
-		t.Fatalf("the container did not produce any output")
-	}
+	assert.Equals(t, len(readBuffer) > 0, true)
```

Comment:
In this case, it's losing the context that it's testing for there being an output. In this case, I would add a comment that says that it means there's no output if the length is zero.